lua/weapons/weapon_lfsmissilelauncher.lua

```diff
@@ -155,9 +159,12 @@ function SWEP:Think()
 			end
 		end
 
-		if self:GetClosestEnt() ~= ClosestEnt then
+		print( findTime, curtime )
+
+		local lockingOnBlockSwitching = ( findTime + ( lockOnTime / 4 ) ) < curtime and IsValid( self:GetClosestEnt() ) and IsValid( ClosestEnt )

```

Comment:
Not sure what this is doing, could this be its own method, or at least split into multiple calculations?

[ref](https://github.com/CFC-Servers/cfc_glua_style_guidelines#complex-expressions-should-be-written-on-multiple-lines-with-meaningful-variable-names)

Reply:
Split it up.